providers/discourse.go

```diff
@@ -0,0 +1,40 @@
+package providers
+
+const Discourse Provider = "discourse"
+
+func init() {
+	SetInfo(Discourse, ProviderInfo{
+		DisplayName: "Discourse",
+		AuthType:    ApiKey,
+		BaseURL:     "https://{{.workspace}}",
```

Comment:
The base url doesn't seem right

Reply:
Discourse is self-hosted, and users can choose their own custom domains for their hosted instances. Since Discourse doesn’t have a fixed base URL, it’s super flexible and can work with any domain.